src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -82,6 +82,7 @@ public static class ScriptConstants
         public const string HttpMethodConstraintName = "httpMethod";
         public const string Snapshot = "snapshot";
         public const string Runtime = "runtime";
+        public const string NugetFallbackFolderRootPath = "D:\\Program Files (x86)\\FuncNuGetFallback";
```

Comment:
Instead of hardcoding the path here, use Environment variable: ProgramFiles(x86)

Reply:
+1